das_client/test_resources/sp/SFERA_SP_9999_1.xml

```diff
@@ -49,25 +52,17 @@
             </TAF_TAP_LocationIdent>
             <TAF_TAP_LocationName language="de" messageString="Bahnhof A"/>
 
-            <!-- todo: Geschwindigkeiten NSP tbd  -->
-
-            <!--            Bahnhofsgeschwindigkeiten -->
-            <TAF_TAP_Location_NSP name="stationSpeed">
-                <NetworkSpecificParameter name="" value="140/[60]"/>
-                <!--            name:    stationSpeed_R150' or 'stationSpeed_A65'. 'stationSpeed_All' or 'stationSpeed_C' 'stationSpeed_R' -->
-                <NetworkSpecificParameter name="graduatedSpeed_00000_n" value=""/>
-                <NetworkSpecificParameter name="graduatedSpeed_00000_ro" value=""/>
-                <NetworkSpecificParameter name="graduatedSpeed_00000_ad" value=""/>
-                <NetworkSpecificParameter name="graduatedSpeed_00000_s" value=""/>
-                <NetworkSpecificParameter name="graduatedSpeed_00000_txt" value="Einfahrt vom Bahnsteig an, übrige Gleise ab Zentralstellwerk"/>
-            </TAF_TAP_Location_NSP>
             <!--            gleisspezifische Geschwindigkeiten	-->
             <TAF_TAP_Location_NSP name="stationProperty_00000">
                 <NetworkSpecificParameter name="stationPropertySpeed_R150" value="30-40"/>
                 <NetworkSpecificParameter name="text" value="&lt;b&gt;31&lt;/b&gt;"/>
             </TAF_TAP_Location_NSP>
         </TAF_TAP_Location>
 
+        <NetworkSpecificArea startEndQualifier="Ends" endLocation="550" name="nonStandardTrackEquipment">
```

Comment:
Gibt es einen Grund, weshalb du dieses NonStandardTrackEquipment wieder hinzugefügt hast? Im 9999 habe ich nur noch eine ETCS Level 2 Strecke gelassen und die Spezialfälle in den T1 verschoben.

Reply:
nein muss wohl ein merge error sein.